src/command_modules/azure-cli-reservations/HISTORY.rst

```diff
@@ -0,0 +1,9 @@
+.. :changelog:
+
+Release History
+===============
+
+0.1.0 (2017-11-06)
```

Comment:
Remove the date. It's no longer required.

Reply:
removed